src/tsc/volar.ts

```diff
@@ -48,6 +49,50 @@ function loadVueLanguageTools() {
   }
 }
 
+function loadEmberLanguageTools() {
+  const debug = Debug('rolldown-plugin-dts:ember')
+  debug('loading ember language tools')
+  try {
+    const emberTscPath = require.resolve('@glint/ember-tsc')
+    const { proxyCreateProgram } = require(
+      require.resolve('@volar/typescript', {
+        paths: [emberTscPath],
+      }),
+    ) as typeof import('@volar/typescript')
+    const ember = require(
+      require.resolve('@glint/ember-tsc', {
+        paths: [emberTscPath],
+      }),
+    ) as typeof import('@glint/ember-tsc')
+    const getLanguagePlugin = (
+      ts: typeof Ts,
+      options: Ts.CreateProgramOptions,
+    ) => {
+      const $rootDir = options.options.$rootDir as string
+      // const $configRaw = options.options.$configRaw as
+      //   | (Ts.TsConfigSourceFile & { vueCompilerOptions?: any })
+      //   | undefined
+
+      const glintConfig = findConfig($rootDir) as GlintConfig
+
+      console.log('GLINT CONFIG', glintConfig)
+
+      // ember.writeGlobalTypes(vueOptions, ts.sys.writeFile)
+      return ember.createEmberLanguagePlugin<string>(
+        // ts,
```

Comment:
@machty Vue allows passing in TS as part of their API. Ember does search it by itself. Maybe not wrong to open up the API to accept TS and only search when it isn't provided?